python/kiss_icp/pipeline.py

```diff
@@ -179,13 +179,22 @@ def _run_evaluation(self):
         # Run timing metrics evaluation, always
         def _get_fps():
             total_time_s = sum(self.times) * 1e-9
-            return float(len(self.times) / total_time_s)
-
-        avg_fps = int(np.ceil(_get_fps()))
-        avg_ms = int(np.ceil(1e3 * (1 / _get_fps())))
+            if total_time_s == 0:
+                return 0  
+            else:
+                return float(len(self.times) / total_time_s)
+
+        fps = _get_fps()
+        if fps == 0:
+            avg_fps = 0
+            avg_ms = 0
+        else:
+            avg_fps = int(np.ceil(fps))
+            avg_ms = int(np.ceil(1e3 / fps))
```

Comment:
```suggestion
        avg_fps = int(np.floor(fps))
        avg_ms = int(np.ceil(1e3 / fps)) if fps > 0 else 0
```

This again saves some lines and reports a more conservative fps and runtime.